edx_exams/apps/router/interop.py

```diff
@@ -23,10 +29,58 @@ def register_exams(course_id, exam_list):
         response = make_request('PATCH', url, client, json=exam_list)
     except HTTPError as e:
         response = e.response
-    return response
 
+    response_data = _get_json_data(response)
+    if response.status_code != status.HTTP_200_OK:
+        log.error(
+            f'Failed to publish exams for course_id {course_id} '
+            f'got status={response.status_code} content={response.content}'
+        )
+
+    return response_data, response.status_code
+
+def get_student_exam_attempt_data(course_id, content_id, user_id):
+    """
+    Get student exam attempt data from the legacy proctoring service
+    """
+    content_id_url_safe = quote_plus(content_id)    # because this goes in the query string
+    path = LMS_PROCTORED_EXAM_ATTEMPT_DATA_API_TPL.format(course_id, content_id_url_safe, user_id)
```

Comment:
The addition of the user_id here requires a minor enhancement to edx-proctoring. Right now this endpoint is only able to look at request.user to determine what user's exam data to get. Since we're tunneling this through edx-exams the user becomes the exams service worker and not the learner.

edx-proctoring needs to support a user_id param that can be used by staff or system users to view another user's exam state.